cmd/exec/exec.go

```diff
@@ -42,15 +43,24 @@ func ExecutePlan(commands ...string) {
 }
 
 func executeAndStopIfFailed(command *models.CommandContext, executeStart time.Time) {
-	command.PrintPhaseBanner()
-	err := command.CommandSession.Run()
-	if err != nil {
+	stdout, err := command.CommandSession.CombinedOutput()
+	if !viper.GetBool("quiet") {
+		command.PrintPhaseBanner()
+		if err != nil {
+			exitCode := (err.Error())[12:]
+			text := "\nExecution failed in phase '" + command.Name + "' – exit code: " + exitCode
+			utils.CPrintln(text, utils.Style{Color: utils.RED})
+			printResultsBanner(false, executeStart)
+			utils.ExitWithCode(exitCode)
+		} else {
+			fmt.Printf("%s", stdout)
+		}
```

Comment:
Test cases are missing for the same. Could you please write test? Let me know if you need any help

Reply:
i will add test cases.